### PR TITLE
chore(release): update misc examples

### PR DESCRIPTION
**Pull request by yT0n1**
Related to CAM-13241

### CODE REVIEW

**Review by tmetzke — COMMENTED** [review]
Looks good in general. Let's try to keep the changes to the standalone files to a minimum. You can have a look at https://github.com/camunda/camunda-bpm-platform/pull/1265 to see what changed from WildFly 20 to 21 in the standalone files.

File: multi-tenancy/schema-isolation/standalone.xml [hunk 1]
```diff
@@ -132,10 +131,10 @@
                 </handlers>
             </root-logger>
             <formatter name="PATTERN">
-                <pattern-formatter pattern="%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %X %s%e%n"/>
+                <pattern-formatter pattern="%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
             </formatter>
             <formatter name="COLOR-PATTERN">
-                <pattern-formatter pattern="%K{level}%d{HH:mm:ss,SSS} %-5p [%c] (%t) %X %s%e%n"/>
+                <pattern-formatter pattern="%K{level}%d{HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
```
Comment (tmetzke) [inline multi-tenancy/schema-isolation/standalone.xml:137]: Those changes are not WildFly update related, I guess?

File: multi-tenancy/schema-isolation/standalone.xml [hunk 2]
```diff
@@ -542,70 +550,71 @@
                         <property name="isAutoSchemaUpdate">true</property>
                         <property name="authorizationEnabled">true</property>
                         <property name="jobExecutorDeploymentAware">true</property>
+                        <property name="historyCleanupBatchWindowStartTime">00:01</property>
```
Comment (tmetzke) [inline multi-tenancy/schema-isolation/standalone.xml:553]: Let's keep the Camunda subsystem as-is, if there is no need to adjust it due to changes in the engine itself.

Comment (tmetzke) [reply multi-tenancy/schema-isolation/standalone.xml:553]: This also applies to the other changes to the subsystem below this one here

**Review by tmetzke — COMMENTED** [review]
I made the points more explicit that don't need a change in my view. Let me know if you think otherwise.

File: multi-tenancy/schema-isolation/standalone.xml (hunk 1 above)
Comment (tmetzke) [reply multi-tenancy/schema-isolation/standalone.xml:137]: If you agree, we can also revert that in the other `standalone.xml`

File: multi-tenancy/schema-isolation/standalone.xml (hunk 2 above)
Comment (tmetzke) [reply multi-tenancy/schema-isolation/standalone.xml:553]: If you agree, we can also revert the changes to that subsystem in the other `standalone.xml`

File: multi-tenancy/schema-isolation/standalone.xml (hunk 1 above)
Comment (yT0n1) [reply multi-tenancy/schema-isolation/standalone.xml:137]: Well, they are Wildfly 19 related. All changes I included are part of the standalone.xml we ship with 7.15 and wildfly 22. I think it makes sense to stick as close as possible to that config, as it is meant to be a drop in replacement.

**Review by tmetzke — APPROVED** [review]
Thanks for the discussion, I agree that we can modify the files like you did to be closer to what we distribute via our WildFly distro so subsequent updates to the examples will be easier to do as well 👍

**Pull request merged**